logic/src/commonMain/kotlin/com/wire/kalium/logic/feature/auth/sso/SSOLoginScope.kt

```diff
@@ -0,0 +1,15 @@
+package com.wire.kalium.logic.feature.auth.sso
+
+import com.wire.kalium.logic.data.auth.login.SSOLoginRepository
+import com.wire.kalium.logic.feature.auth.ValidateUUIDUseCase
+import com.wire.kalium.logic.feature.auth.ValidateUUIDUseCaseImpl
+
+class SSOLoginScope(
+    private val ssoLoginRepository: SSOLoginRepository
+) {
+    private val validateUUIDUseCase: ValidateUUIDUseCase get() = ValidateUUIDUseCaseImpl()
+    val initiate: SSOInitiateLoginUseCase get() = SSOInitiateLoginUseCaseImpl(ssoLoginRepository, validateUUIDUseCase)
```

Comment:
For this one is suggest `SSOLoginInitiatorUseCase`, what do you think?

Reply:
Same answer as above about "validator", also the endpoint is `/sso/initiate-login` so it's easier to understand for me at least if the use case has very similar name